src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -102,6 +104,7 @@ def default_api_version(self):
         ResourceType.MGMT_RESOURCE_POLICY: '2015-10-01-preview',
         ResourceType.MGMT_RESOURCE_RESOURCES: '2016-02-01',
         ResourceType.MGMT_RESOURCE_SUBSCRIPTIONS: '2016-06-01',
+        ResourceType.MGMT_AUTHORIZATION: '2015-07-01',
```

Comment:
Replace by:
```python
ResourceType.MGMT_AUTHORIZATION: SDKProfile('2015-07-01')
```
This implies not classic_administrators support in Stack. If it's not true, add the classic_admin admin key like I did above